tests/test_gplvm.py

```diff
@@ -22,7 +21,6 @@
 from gpflow.utilities.ops import pca_reduce
 
 
-@dataclass
```

Comment:
Why did you remove the `dataclass`?

Reply:
Because the code works exactly the same without, and then I don't need the extra dependency. Why would it have to be a `@dataclass`?